Datadog/Example/Debugging/DebugRUMSessionViewController.swift

```diff
@@ -39,6 +39,12 @@ private class DebugRUMSessionViewModel: ObservableObject {
     @Published var errorMessage: String = ""
     @Published var resourceKey: String = ""
 
+    @Published var logMessage: String = ""
+    @Published var spanOperationName: String = ""
+    @Published var instrumentedRequestURL: String = "https://api.shopist.io/checkout.json"
```

Comment:
currently it returns 
```json
{
    "status": 404,
    "error": "Not Found"
}
```

expected ?

Reply:
It is `POST` endpoint (`GET` always returns `404`), but getting some % of errors on it is expected as this is part of demo environment 👍.